embedded-graphics/CHANGELOG.md

```diff
@@ -6,6 +6,16 @@ Embedded Graphics is a `no_std` library for adding graphics features to display
 
 ## [Unreleased] - ReleaseDate
 
+## [0.7.0-alpha.1] - 2020-09-19
+
+### Added
+
+- [#386](https://github.com/jamwaffles/embedded-graphics/pull/386) Added `Line::midpoint` to get the midpoint of a line.
+- [#386](https://github.com/jamwaffles/embedded-graphics/pull/386) `Polyline` now supports stroke widths greater than 1. Stroke alignment is not supported.
```

Comment:
Should we mention the stroke alignment in the changelog? This could be interpreted as a limitation of the current version which will be fixed later, but not supporting this was a deliberate choice.